ui/desktop/app/components/hidden-secret/index.hbs

```diff
@@ -11,12 +11,12 @@
     {{t 'actions.toggle-secret'}}
   </Rose::Button>
   <Copyable
-    @text={{@secret}}
+    @text={{this.displaySecretValue}}
     @buttonText={{t 'actions.copy-to-clipboard'}}
     @acknowledgeText={{t 'states.copied'}}
   >
     {{#if this.isHidden}}
-      {{@secret}}
+      <pre>{{this.displaySecretValue}}</pre>
```

Comment:
We only need the `<pre>` when displaying an object. It is messing with the styling when displaying plain text.
<img width="626" alt="Screen Shot 2023-01-03 at 11 57 51 AM" src="https://user-images.githubusercontent.com/29386339/210415310-23a6e8c9-133f-426f-83dc-7ac003d6cd3d.png">

Reply:
We still need to address this issue. `<pre>` is causing plain text secrets to render off centered.